docs/coding-guidlines.md

```diff
@@ -0,0 +1,78 @@
+# Coding Guidelines
+
+Use the following coding guidelines when creating and editing players:
+
+## Naming
+
+Use the following naming conventions:
+
+
+| Convention        |Explanation | 
```

Comment:
I think it makes sense to have these in a table - if you are OK with that...

Reply:
defiantly

Oded